data-prepper-logstash-configuration/src/main/java/com/amazon/dataprepper/logstash/model/Attribute.java

```diff
@@ -0,0 +1,22 @@
+package com.amazon.dataprepper.logstash.model;
+
+public class Attribute {
```

Comment:
The `Attribute` class is already overloaded in many places. Can we rename this to `LogstashConfigurationAttribute`?

Reply:
Yeah, I'll change all the class names.

Reply:
Would we be fine with simply `LogstashAttribute`? The pattern could be to prepend `Logstash` to the class names, thus `LogstashClassName`. This is a little less verbose than `LogstashConfigurationClassName`.

Reply:
I'm fine with that.